TOC/README.md

```diff
@@ -0,0 +1,88 @@
+# TiDB Community Technical Oversight Committee (TOC)
+
+The TiDB Community TOC serves as the main bridge and channel for coordinating and information sharing across companies and organizations. It is the coordination center for solving problems in terms of resource mobilization, technical research and development direction in the current community and cooperative projects.
+
+## Responsibilities:
+
+- Partner cooperation project progress monitoring and guidance
+- Supervision and guidance on the progress of the TiDB ecological project
+- Participate in technical route discussion and decision-making, such as version release and roadmap formulation, etc.
+- Open source collaboration process discussion and decision
+- Discussion and decision-making of community-related activities
+
+## Membership
+
+TOC members include official members and non-voting members:
+
+- Official members have the right to participate in TOC meetings, propose proposals, and vote.
+- Non-voting members have the right to participate in TOC meetings, propose proposals, but do not have the right to vote.
+
+TOC members come from community partners who have signed a cooperation agreement with PingCAP and representatives of important community projects. Each community partner has 1 official member seat. TiDB, TiKV and Chaos Mesh®  each have 1 official member seat. The seats of non-voting members are not limited.
+
+## Nomination
+
+All official members of TOC have the right to nominate representative as new TOC member. The nominee is required to meet the following standards:
+
+- The nominee must have enough energy to invest in TOC affairs, such as participating in TOC discussions and [decision making](./Proposal), and participating in TOC monthly meetings.
+- The nominee must have a deep understanding of TiDB or related ecological technologies. The nominee is at least the Committer of TiDB/TiKV/Chaos Mesh®  or TiDB ecological project.
```

Comment:
After having a second thought about this particular standard, I suggest we eliminate the prerequsite that the nominee must be a committer. The rationale behind this suggestion are: 

1. Members of TOC represents the organization that have key impact to the community, but not for themselves.
2. Their responsibilities are mainly to coordinate resource investment and make decisions for long-term goals. These responsibilities do not require writing lots of code but having greater vision about the TiDB community.

Reply:
Yes, you convinced me. If no one objects, I will eliminate the committer prerequisite.